src/kms.py

```diff
@@ -0,0 +1,7 @@
+import boto3
+
+def decrypt(encrypted_key):
+    kms_client = boto3.client('kms')
+    response = kms_client.decrypt(CiphertextBlob=encrypted_key)
+    print("Response: ", response)
```

Comment:
I don't think we should be printing/logging the decrypted key - we should assume that there might be readonly users who might see logs who shouldn't have access to this kind of data.

Reply:
I missed it when I was debugging the Python script yesterday. I have removed it.